### ISSUE TITLE
Resized image: no offer to download file if zooming

### ISSUE THREAD

**Issue by tobiasKaminsky** [nextcloud/android#2604]
With https://github.com/nextcloud/android/pull/2593 this does not work any more:
- open up a non-downloaded image in preview
- zoom into it
- after 2x zoom previously it showed "download full image?"
- if no internet connection is available it also showed some info

--> I guess that only a zooming listener is missing.

@ardevd can you have a look into this? As always I can help you, if something is not clear.

**Comment by ardevd** [comment]
Hi. 

Me and @AndyScherzinger tested this yesterday and I cant see any difference between the behavior before and after #2593. 

Tapping to zoom doesnt produce the "download full image" message either way. However, pinch-to-zoom a bit and you'll get it. 

If I understand you correctly, you'd want to show that message when tapping to zoom as well. Again, this doesnt happen when I use Nextcloud app from Google Play either but I agree that it would be a nice feature.

Once I get a better understanding on what behaviour is expected I'd be happy to look at it.

**Comment by ardevd** [comment]
Also, I believe this is a duplicate of #2598?

**Comment by tobiasKaminsky** [comment]
@ardevd right, I always tried it with pinch-to-zoom and not double click, sorry for the confusion :+1:

**Comment by tobiasKaminsky** [comment]
And yes, it is a duplicate.

**Comment by AndyScherzinger** [comment]
see https://github.com/nextcloud/android/issues/2598#issuecomment-391154156 it seems it doesn't work anymore since today, but I am sure it worked yesterday...
